### PR TITLE
[13.0][FIX] sale_product_set: Change toggle_active button to web_ribbon widget in form view.

### PR DESCRIPTION
**Pull request by victoralmau**
Change `toggle_active` button to `web_ribbon` widget in form view.

Please @pedrobaeza and @CarlosRoca13 can you review it?

@Tecnativa TT34555

### CODE REVIEW

File: sale_product_set/wizard/product_set_add.py [hunk 1]
```diff
@@ -10,10 +10,12 @@ class ProductSetAdd(models.TransientModel):
     _rec_name = "product_set_id"
     _description = "Wizard model to add product set into a quotation"
 
+    required_field = fields.Selection(
```
Comment (pedrobaeza) [inline sale_product_set/wizard/product_set_add.py]: Don't do this on your inherited module, but inherit the wizard model with other name, and there, put `order_id` as required=False, and the new m2o as required=True

**Review by pedrobaeza — APPROVED** [review]

**Review by CarlosRoca13 — APPROVED** [review]
LGTM :+1:

**Comment by pedrobaeza** [comment]
Cherry-pick to v14 at b945fe4560fd539cedfd95c0a2fca428fb3492f5

**Pull request merged**